corehq/motech/generic_inbound/models.py

```diff
@@ -72,3 +76,50 @@ def to_json(self):
             "expression_id": self.expression_id,
             "message": self.message,
         }
+
+
+class RequestLog(models.Model):
+
+    class Status(models.TextChoices):
+        FILTERED = 'filtered', _('Filtered')
+        VALIDATION_FAILED = 'validation_failed', _('Validation Failed')
+        SUCCESS = 'success', _('Success')
+        ERROR = 'error', _('Error')
+        REVERTED = 'reverted', _('Reverted')
+
+    class RequestMethod(models.TextChoices):
+        POST = 'POST'
+        PUT = 'PUT'
+        PATCH = 'PATCH'
```

Comment:
I considered moving this to `motect/consts.py`, but ended up reasoning that a use-case-specific set of choices might make more sense.